core/modules/update/theme/update.theme.inc

```diff
@@ -78,10 +78,13 @@ function theme_update_report($variables) {
   }
 
   foreach ($data as $project) {
+    //Don't show a link to the "Update modules" page for core
+    $updatelink = ($project['name'] != 'backdrop');
```

Comment:
Coding standards: We always separate words within a variable by an underscore, e.g. `$update_link`